File: Domains/ApplicationDomain/Common/TestedResult.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace ApplicationDomain.Common
{
    public class TestedResult
    {
        public float Value { set; get; }
        public float Max { set; get; }
        public float Min { set; get; }
        public string TestedEvaluation
        {
            get
            {
                return this.Value < this.Min ? Enum.GetName(typeof(Evaluation), Evaluation.Under) :
                    this.Value > this.Min && this.Value < this.Max ? Enum.GetName(typeof(Evaluation), Evaluation.Normal) :
                    this.Value < this.Max * 1.1 ? Enum.GetName(typeof(Evaluation), Evaluation.SlightlyOver) :
                    Enum.GetName(typeof(Evaluation), Evaluation.Over);
            }
        }
    }
}

File: Applications/ClientApp/src/app/inbody/core/models/tested-result.ts
export enum Evaluation {
  Normal,
  Under,
  SlightlyOver,
  Over,
}

export class TestedResult {
  max: number;
  min: number;
  value: number;

  constructor() {
  }

  get testedEvaluation() {
    return this.value < this.min ? Evaluation[Evaluation.Under] :
      this.value > this.min && this.value < this.max ? Evaluation[Evaluation.Normal] :
        this.value < this.max * 1.1 ? Evaluation[Evaluation.SlightlyOver] :
          Evaluation[Evaluation.Over];
  }
}
